File: pkg/pwrgrep/testdata/fixtures/java/Variables.java
public class Variables {
    // ruleid: java-variables
    private static final int LIMIT = 10;
    // ruleid: java-variables
    private String name;

    void f() {
        // ruleid: java-variables
        var list = new java.util.ArrayList<String>();
        // ruleid: java-variables
        int x = 1;
        // ok: java-variables
        x = 2;
    }
}

File: pkg/pwrgrep/testdata/fixtures/go/variables.go
package fixture

// ruleid: go-variables
const limit = 10

// ruleid: go-variables
var global = 3

func f() int {
	// ruleid: go-variables
	x := 1
	// ruleid: go-variables
	var y int
	// ok: go-variables
	y = x
	// ruleid: go-variables
	a, b := 1, 2
	return y + a + b + global + limit
}
